documentation/src/main/asciidoc/stories/assembly_query_creating_ickle_queries.adoc

```diff
@@ -6,8 +6,10 @@
 include::{topics}/ref_query_ickle.adoc[leveloffset=+1]
 include::{topics}/ref_query_ickle_syntax.adoc[leveloffset=+1]
 include::{topics}/ref_query_full_text_search.adoc[leveloffset=+1]
+ifdef::community[]
 include::{topics}/ref_query_vector_search.adoc[leveloffset=+1]
-
+endif::[]
+include::{topics}/ref_query_spatial_search.adoc[leveloffset=+1]
```

Comment:
is spatial search supported in 15.0!? cc @fax4ever @theashiot @tristantarrant

Reply:
IIRC we decided to not backport it